lib/v2/tfbd/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'tfbd.ccf.org.cn': {
```

Comment:
```suggestion
    'ccf.org.cn': {
```
This should include the root domain only